src/cargo/ops/cargo_add/mod.rs

```diff
@@ -518,8 +531,10 @@ fn get_existing_dependency(
 }
 
 fn get_latest_dependency(
+    spec: &Package,
     dependency: &Dependency,
     _flag_allow_prerelease: bool,
```

Comment:
side note: no clue what this flag is, doesn't appear to ever have been used in cargo. I'm guessing a holdover from cargo-edit?

Reply:
Yeah, that is a vestige of an old cargo-edit version of cargo-add and can be removed.